lib/trino-parquet/src/main/java/io/trino/parquet/writer/ParquetWriterOptions.java

```diff
@@ -35,8 +34,8 @@ public static ParquetWriterOptions.Builder builder()
 
     private ParquetWriterOptions(DataSize maxBlockSize, DataSize maxPageSize, int batchSize)
     {
-        this.maxRowGroupSize = toIntExact(maxBlockSize.toBytes());
-        this.maxPageSize = toIntExact(maxPageSize.toBytes());
+        this.maxRowGroupSize = Ints.saturatedCast(maxBlockSize.toBytes());
```

Comment:
We could add the below annotations in ParquetWriterConfig for `parquet.writer.block-size`
```
@MinDataSize("8MB")
@MaxDataSize("2GB")
```